spec/models/condition_spec.rb

```diff
@@ -1,5 +1,26 @@
 describe Condition do
   describe ".subst" do
+    context "evaluation of virtual custom attributes from left and right side" do
+      let(:custom_attribute_1)         { FactoryGirl.create(:custom_attribute, :name => "attr_1", :value => 20) }
+      let(:custom_attribute_2)         { FactoryGirl.create(:custom_attribute, :name => "attr_2", :value => 30) }
+      let(:name_of_custom_attribute_1) { "VmOrTemplate-#{CustomAttributeMixin::CUSTOM_ATTRIBUTES_PREFIX}attr_1" }
```

Comment:
Not a problem though, it's probably better for the test to make this explicit :+1: